app/pages/Accounts/AccountViewActions/AccountViewActions.tsx

```diff
@@ -36,12 +33,12 @@ const receiveAction: ActionObject = {
     height: 22,
     isDisabled: () => false,
 };
-const shieldedActions: ActionObject[] = [
+
+const unshieldedActions: ActionObject[] = [
```

Comment:
Maybe just 
```suggestion
const actions: ActionObject[] = [
```

As these don't change anymore regardless of which balance is being displayed